chronos/asyncfutures2.nim

```diff
@@ -36,8 +36,9 @@ type
     stackTrace: StackTrace ## For debugging purposes only.
     mustCancel*: bool
     id*: int
-    next*: FutureBase
-    prev*: FutureBase
+    when defined(chronosFutureTracking):
```

Comment:
shouldn't this cover the stack trace as well? it's a pure debugging feature, and one of the most expensive things nbc does, according to profiling, when not syncing

Reply:
stack trace is not related to `chronosFutureTracking`.

Reply:
yeah, but maybe it should be, naming this feature `chronosDebug` or something like that?

Reply:
`chronosStackTraces`.